### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t2051

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: css/editors.less [hunk 1]
```diff
@@ -145,6 +146,28 @@
...
+        margin-right: 15px;
+        display: inline-block;
+        cursor: pointer;
+        border-right: 1px solid @baseLightOutline;
+        .triangle( "right", 5px, #FFF, 1px, @baseLightOutline  );
+        &:hover {
+          opacity: 1;
+        }
+      }
+      .close-btn{
```
Comment (k88hudson) [inline css/editors.less]: spacing

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e77ce322b111fb3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e77ce322b111fb3/output.txt

Total script time: 0.79 mins

Server is running on http://pr824.butyr.org/templates/basic/
Run the Test Suite: http://pr824.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/198dbe58edc1bb9/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/198dbe58edc1bb9/output.txt

Total script time: 1.08 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/3d65f902b2ba3d6/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/3d65f902b2ba3d6/output.txt

Total script time: 0.60 mins

Server is running on http://pr824.butyr.org/templates/basic/
Run the Test Suite: http://pr824.butyr.org/test/

File: src/editor/trackevent-editor.js [hunk 2]
```diff
@@ -92,12 +91,38 @@ define( [ "util/lang", "util/keys", "./base-editor",
...
 
     extendObject.defaultLayouts = __defaultLayouts.cloneNode( true );
 
+    extendObject.createTitle = function( trackEvent, editorModule ) {
+      var titleEl = rootElement.querySelector( "h1" ),
+          addPopcornLink = document.createElement( "a" ),
+          closeEditorLink = document.createElement( "a" );
+
+      closeEditorLink.classList.add( "close-btn" );
+      closeEditorLink.innerHTML = '<span class="icon icon-only icon-x"></span>';
```
Comment (mjschranz) [inline src/editor/trackevent-editor.js]: Change to double quotes then escape the other double quotes.

File: css/globals.less [hunk 3]
```diff
@@ -185,3 +185,28 @@
...
+    border-left-color: @color;
+    border-width: @size;
+    top: 50%;
+    margin-top: -@size;
+  }
+  &:before {
+    border-left-color: @borderColor;
+    border-width: @size + @borderSize + 1;
+    top: 50%;
+    margin-top: -( @size + @borderSize + 1);
```
Comment (mjschranz) [inline css/globals.less]: space right bracket.

File: css/editors.less [hunk 4]
```diff
@@ -145,6 +144,28 @@
...
       .box-sizing( border-box );
+      .butter-breadcrumbs {
+        opacity: 0.6;
+        position: relative;
+        padding-right: 15px;
+        margin-right: 15px;
+        display: inline-block;
+        cursor: pointer;
+        border-right: 1px solid @baseLightOutline;
+        .triangle( "right", 5px, #FFF, 1px, @baseLightOutline  );
```
Comment (mjschranz) [inline css/editors.less]: One too many spaces before right bracket

File: src/editor/trackevent-editor.js [hunk 5]
```diff
@@ -92,12 +91,38 @@ define( [ "util/lang", "util/keys", "./base-editor",
...
+      closeEditorLink.addEventListener( "click", function( e ) {
+        editorModule.openEditor( "plugin-list" );
+      }, false);
+
+      addPopcornLink.classList.add( "butter-breadcrumbs" );
+      addPopcornLink.innerHTML = "Events";
+      addPopcornLink.addEventListener( "click", function( e ) {
+        editorModule.openEditor( "plugin-list" );
+      }, false);
```
Comment (mjschranz) [inline src/editor/trackevent-editor.js]: Space right bracket

File: src/editor/trackevent-editor.js [hunk 6]
```diff
@@ -92,12 +91,38 @@ define( [ "util/lang", "util/keys", "./base-editor",
...
+    extendObject.createTitle = function( trackEvent, editorModule ) {
+      var titleEl = rootElement.querySelector( "h1" ),
+          addPopcornLink = document.createElement( "a" ),
+          closeEditorLink = document.createElement( "a" );
+
+      closeEditorLink.classList.add( "close-btn" );
+      closeEditorLink.innerHTML = '<span class="icon icon-only icon-x"></span>';
+      closeEditorLink.addEventListener( "click", function( e ) {
+        editorModule.openEditor( "plugin-list" );
+      }, false);
```
Comment (mjschranz) [inline src/editor/trackevent-editor.js]: space right bracket

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/0c0de2e4ba83036/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/0c0de2e4ba83036/output.txt

Total script time: 1.18 mins

Server is running on http://pr824.butyr.org/templates/basic/
Run the Test Suite: http://pr824.butyr.org/test/

**Comment by jbuck** [comment]
Bobby's been assigned this ticket, so closing this pull request

**Pull request closed without merging**